### PR TITLE
[hist] explain better what computeIntegral does

### PR DESCRIPTION
**Pull request by ferdymercury**
# This Pull request:

## Changes or fixes:

Fixes https://its.cern.ch/jira/browse/ROOT-8274

## Checklist:

- [ ] tested changes locally
- [x] updated the docs (if necessary)

### CODE REVIEW

**Comment by dpiparo** [comment]
thanks a lot for this PR!

**Comment by dpiparo** [comment]
@lmoneta can we merge?

**Comment by dpiparo** [comment]
@ferdymercury could you clamg-format your code? Then I think we are ready to merge.

**Comment by ferdymercury** [comment]
> @ferdymercury could you clamg-format your code? Then I think we are ready to merge.

Sure, done.

File: hist/hist/src/THnBase.cxx [hunk 1]
```diff
@@ -1319,14 +1325,14 @@ Double_t THnBase::ComputeIntegral()
 
    // check sum of weights
    if (fIntegral[GetNbins()] == 0.) {
-      Error("ComputeIntegral", "No hits in regular bins (non over/underflow).");
+      Error("ComputeIntegral", "Integral = 0, no hits in regular bins (non over/underflow).");
```
Comment (vepadulano) [inline hist/hist/src/THnBase.cxx:1328]:
```suggestion
      Error("ComputeIntegral", "Integral = 0, no hits in histogram bins (excluding over/underflow).");
```

**Review by vepadulano — APPROVED** [review]
Great, many thanks! A couple of nitpicks, feel free to consider or not

**Pull request merged**